media-gfx/blender/blender-2.79b.recipe

```diff
@@ -205,7 +204,7 @@ BUILD()
 
 	# Release build settings
 	BLENDER_RELEASE="-DWITH_ALEMBIC=ON \
-		-DWITH_BUILDINFO=ON \
+		-DWITH_BUILDINFO=OFF \
```

Comment:
What does this? Did they changed the release-settings? If not, then it have to be moved to the Haiku specific settings. The settings separated to be able to follow, where the Haiku port diverges from the official builds.

Reply:
Here is the release spec: https://github.com/blender/blender/blob/master/build_files/cmake/config/blender_release.cmake#L9

Reply:
This disables displaying build info in splash screen. It depends on git executable.

Reply:
Makes sense. Could you please move it to the Haiku specific options?

Reply:
Done.